spec/text-editor-component-spec.coffee

```diff
@@ -601,6 +601,48 @@ describe "TextEditorComponent", ->
       expect(lineNode.offsetTop).toBe(top)
       expect(lineNode.textContent).toBe(text)
 
+    it "renders tiles upper in the stack in front of the ones below", ->
```

Comment:
The phrase "upper in the stack" sounds a bit unusual. I'd maybe say "it renders higher tiles in front of lower ones".